src/Events/Index.jsx

```diff
@@ -0,0 +1,284 @@
+  overflow: hidden;
+
+  @media (max-width: 900px) {
+    padding: var(--section-gap) 24px;
+  }
+`;
+
+const Container = styled.div`
+  display: flex;
+  max-width: 1224px;
+  margin: 0 auto;
+  gap: ${(p) => p.gap ?? "var(--section-gap)"};
+  flex-direction: column;
+  align-items: ${(p) => (p.center ? "center" : undefined)};
+  justify-content: ${(p) => (p.center ? "center" : undefined)};
+  text-align: ${(p) => (p.center ? "center" : undefined)};
+`;
+
+const Pattern = styled.div`
+  width: 100%;
+  min-height: 540px;
+  display: flex;
+  align-items: center;
+  background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAJYAAACWCAYAAAA8AXHiAAAACXBIWXMAABYlAAAWJQFJUiTwAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAGeSURBVHgB7doxTisxEAbgeY/mvQro6NiSDo6QkpJbcA2OwjWooKQMJ2DpKENJBV7FEYoBeQSIZr9PGk2cItWvsdfZnSBjKHVf6rnUbdD1N8g4K7VX6jhIEaycofaTIEWwcoam0yFYOYe179WiQ7Byhk8+8wnB6munlHNWgmD1tUGyFSYIVl8bJFcOCYLV106s/aBrJ2hNE+qo1GmpRanz2J5aB6X+x/oQv/l+FWz5E/O1iHU4pom0W/u0/uoZahnrgN2VGuv6Jpidl1+o2T5BznkrfKj9MdZT6l9836r+3k2pq1KXMVNz3gpbU7hOmj49AQ7x/lJ0WWsK5xhv2+AYkHQR29vbddDluqFvbNZPQZdg9S07az4gWH3tHZVgJQhW3xjb4XIZyo+Z3nffHN79CZ1gYuXc1b4KEytFsHLGptMhWDlj7Q9BimDlbJ4Ex4AftggHdwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIpXoUVLSWulnzoAAAAASUVORK5CYII=");
+  background-size: 75px 75px;
+  background-repeat: repeat;
+  background-position: center top;
+
+  @media (max-width: 900px) {
+    min-height: 390px;
+  }
+`;
+
+const PatternContent = styled.div`
+  padding: 1rem;
+  max-width: 496px;
+  margin: 0 auto;
+  background-color: var(--background-color);
+  display: flex;
+  align-items: center;
+  min-height: 260px;
+
+  @media (max-width: 900px) {
+    min-height: 0px;
+  }
+`;
+
+const IconCircle = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 64px;
+  height: 64px;
+  border-radius: 100%;
+  border: 1px solid var(--sand11);
+
+  i {
+    color: var(--sand11);
+    font-size: 32px;
+  }
+`;
+
+const featuredEvent = [...eventsList].sort(
+  (a, b) => Math.abs(new Date(a.event.start_at) - new Date()) - Math.abs(new Date(b.event.start_at) - new Date()),
+)[0].event;
+
+return (
+  <Wrapper>
+    <Section backgroundColor="#fff" style={{ padding: "72px 0" }}>
+      <Container center>
+        <Pattern>
+          <PatternContent>
+            <Flex gap="32px" direction="column" alignItems="center">
+              <H1>Events</H1>
+
+              <Text size="text-l" mobileSize="text-base">
+                Come together IRL and online, and be part of building the Open Web together.
+              </Text>
+            </Flex>
+          </PatternContent>
+        </Pattern>
+      </Container>
+    </Section>
+
+    <Section backgroundColor="#fff" style={{ padding: "72px 24px" }}>
+      <Container>
+        <Flex direction="column" gap="80px" mobileGap="40px">
+          <Text size="text-3xl" mobileSize="text-2xl" fontWeight="500">
+            Upcoming Event
```

Comment:
As far as we don't have a mechanism to set the featured event we'll show the upcoming event